apps/src/templates/SchoolZipSearch.jsx

```diff
@@ -53,6 +69,7 @@ export default function SchoolZipSearch({fieldNames}) {
             styleAsText={true}
             onClick={() => {
               setInputManually(false);
+              setSelectedSchoolNcesId('selectASchool');
```

Comment:
nit: could this be moved into its own constant, used both here and in `SEARCH_DEFAULTS`?